config/longrun_configs/gpu_amip_target_topo_diagedmf.yml

```diff
@@ -1,26 +1,30 @@
 FLOAT_TYPE: "Float32"
+albedo_model: "CouplerAlbedo"
 anim: false
 apply_limiter: false
-atmos_config_file: "config/longrun_configs/longrun_aquaplanet_rhoe_equil_55km_nz63_allsky_diagedmf_0M.yml"
-dt: "100secs"
+atmos_config_file: "config/gpu_configs/gpu_aquaplanet_diagedmf.yml"
+dt: "120secs"
 dt_cloud_fraction: "1hours"
-dt_cpl: 100
+dt_cpl: 120
 dt_rad: "1hours"
 dt_save_state_to_disk: "20days"
 dt_save_to_sol: "10days"
 energy_check: false
-hourly_checkpoint: true
+hourly_checkpoint: false
 idealized_insolation: false
 job_id: "gpu_amip_target_topo_diagedmf"
 land_albedo_type: "map_temporal"
 mode_name: "amip"
 mono_surface: false
 netcdf_output_at_levels: true
+output_default_diagnostics: true
+rayleigh_sponge: true
 run_name: "gpu_amip_target_topo_diagedmf"
 start_date: "19790301"
 surface_setup: "PrescribedSurface"
-t_end: "90days"
+t_end: "250days"
```

Comment:
Do we want this to be 300days, or is 250 enough?

Reply:
Yes, with the increased timestep this should be possible, even with the diagnostics switched on.